### DISCUSSION TITLE
How to buy new ISIC Timelord?

### DISCUSSION THREAD

**Discussion by adspx5** [Chia-Network/chia-blockchain#17678] [Q&A]
Who knows how to purchase recently announced ISIC Timelord?
https://www.chia.net/2023/10/26/asic-timelords-faster-than-fast-chia-network/

Thank you!

**Comment by BrandtH22** [comment]
Hey @adspx5 , unfortunately the application period for timelords has ended. There might be the opportunity to purchase a timelord at a later date from CNI or from a third party resale but that timeline is yet to be determined.

Do let us know if any questions arise but since we are not opening the applications for additional applicants at this time I will close out this discussion for now.
